src/i18n/content/jp/docs/kubernetes-pixie/kubernetes-integration/get-started/introduction-kubernetes-integration.mdx

```diff
@@ -13,7 +13,7 @@ Kubernetesは、コンテナ化されたアプリケーションのデプロイ
 
 [KubernetesKubernetes インテグレーション](/docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure/) し たら、Kubernetes のすぐに使用できるエクスペリエンスを活用して、 上で実行されているすべてのサービスをKubernetes 1[Kubernetes ](/docs/integrations/kubernetes-integration/cluster-explorer/kubernetes-cluster-explorer)か所で監視できるようになります。コントロール プレーンの最上部から単一のポッドで実行されているアプリケーションに至るまですべてが表示されるため、クラスターの動作の核心に迫ることができます。
```

Comment:
Marking this just in case!!! Translation

Reply:
This change doesn't affect the content itself.